### ISSUE TITLE
Breadcrumb Style not compatible/accessible with Magister Theme

### ISSUE THREAD

**Issue by MDWSanoma** [sl-design-system/components#1348]
### Provide a general summary of the issue here

The Breadcrumb style in Magister Themes are not compatible with the Designs of Magister Screens. We're now using a page header with a dark background, and we'd like the breadcrumbs to be white, not light blue and primary blue as it is now in Dark mode.

### 🤔 Expected Behavior?

![Screenshot 2024-06-18 at 10 56 30](https://github.com/sl-design-system/components/assets/121878399/9ba77c78-5d9a-4d12-8fc9-3b9df55f5fd9)

Option for high-contrast (white) text for the text

### 😯 Current Behavior

Magister Light:

![Screenshot 2024-06-18 at 10 57 43](https://github.com/sl-design-system/components/assets/121878399/4ff9e7f2-655f-447a-a2ed-76a492abe129)

Magister Dark:

![Screenshot 2024-06-18 at 10 58 51](https://github.com/sl-design-system/components/assets/121878399/52915339-5556-4de9-b99f-b66f39c7b592)

Neither of these are compatible, or accessible.

(There's also a weird bug with the first level, styling is off)

### 💁 Possible Solution

Option for a high contrast (white) version for the page header designs with a dark background. We'd still need a version as it is now for the current screens that have a white page header (no background)

See: https://www.figma.com/design/o1ZlCUh6LyicB6xBu2GRmC/branch/XDZXzzzZYkLskuT8BM5nbE/DNA-Library?m=auto&node-id=0-1623&t=H7MDq4KSp6ot9s46-1 (We're now still using our own breadcrumb component)

### 📎 Link to Figma

https://www.figma.com/design/YZeBneflpJkX3KXSyvMrqP/Magister-Light-mode?m=auto&node-id=1765-1464&t=CXAqrzMPwmtfVQ2o-1

### 🎨 Your Theme(s)

Magister

**Comment by DanielleRameau** [comment]
@MDWSanoma Roald is currently not here, we will discuss this topic next week during the refinement together with him. This issue doesn't only affect the breadcrumbs but other components as well so we have to implement this 'on dark' feature within our epics 'provide more context in tokens'.

**Comment by DanielleRameau** [comment]
@MDWSanoma we created a new story to take a look at a ondark property, see #1405 it's on the backlog. Once that is ready we can work on this.
